src/definitions.ts

```diff
@@ -718,6 +719,21 @@ export interface PurchasesPlugin {
     storeProduct: PurchasesStoreProduct;
   }): Promise<{ refundRequestStatus: REFUND_REQUEST_STATUS }>;
 
+  /**
+   * Shows in-app messages available from the App Store or Google Play. You need to disable messages from showing
+   * automatically using [PurchasesConfiguration.shouldShowInAppMessagesAutomatically].
+   *
+   * Note: In iOS, this requires version 16+. In older versions the promise will be resolved successfully
+   * immediately.
+   *
+   * @param options An array of message types that the stores can display inside your app. Values must be one of
```

Comment:
```suggestion
   * @param messageTypes An array of message types that the stores can display inside your app. Values must be one of
```

Reply:
Actually, the parameter here is `options`. We could rename it to `messageTypes` since the object passed in only has that but shouldn't really matter.

Reply:
ohh I see what you mean, nvmd I reviewed this too late lol